docs/demo-data/pull-request-usage.tsv

```diff
@@ -1,4 +1,4 @@
-date	Pull request usage in %	Status usage in %
+date	pull request usage [%]	status usage [%]
```

Comment:
Is this some kind of standard? I think `in %` reads easier, no?

Reply:
Yep, that’s how you usually denote the unit of an axis. I think it looks more professional like this.

Reply:
Also keep in mind that I want to show the unit on the axes directly at some point. Parsing might be easier with this bracket notation.

Reply:
parsing is a convincing argument! 👍